src/main/java/com/easypost/net/Requestor.java

```diff
@@ -48,7 +48,7 @@
 import java.util.Scanner;
 
 public abstract class Requestor {
-    protected enum RequestMethod {
+    public enum RequestMethod {
```

Comment:
Issue: this should remain `protected`. Users will never need access to this.

Reply:
Since we are no longer extending `Requestor` in `EasyPostResource`, we need to make this `public` so we could use this enum class in the `request` functions in every class.

Reply:
I thought the whole point of protected was that anything inside our own library could use it?

Reply:
No Justin😂, `protected` means the subclass can use the property from the parent class, but since we are no longer extending to the `Requestor` class in the `EasyPostResource` file, we just lost the access permission for it. 

Link to a resource: https://www.geeksforgeeks.org/protected-keyword-in-java-with-examples/

Reply:
Our entire library uses `package com.easypost.net;`. Based on the link you sent me, we should still be able to use `protected` here because even though we no longer inherit these, all of this functionality still lives inside the same package.

Reply:
No, we can't. They are in different packages. `Requestor` is in `com.easypost.net` while our classes are in `package com.easypost.model`.

Reply:
Java is a terrible language.

Reply:
I'd recommend using the "default" access modifier instead, so the function isn't open to end-users, just inside our library.

Reply:
We can't use `default` since they are in different packages, please let me know which function(s) you're referring to

Reply:
I just made a small tweak, moving the `RequestMethod` enum to EasyPostResource and then importing it in the `Requestor` file since they are in the package. This will avoid end-users from using this property.